core/proofs/impl/proof_engine_impl.cpp

```diff
@@ -38,6 +38,11 @@ namespace fc::proofs {
   using primitives::sector::RegisteredAggregationProof;
   using sector_storage::zerocomm::getZeroPieceCommitment;
 
+  inline auto rand31(PoStRandomness rand) {
+    rand[31] &= 0x3f;
+    return rand;
+  }
+
```

Comment:
If it is an operation under the PoStRandomness, why not to modify it via inner class function f.e. self modified getter

Reply:
This function is only used in this file.  
Currently `PoStRandomness` is alias to 32 bytes, so we can't add member function.